pxr/imaging/hd/instanceRegistry.h

```diff
@@ -194,7 +194,7 @@ class HdInstanceRegistry {
 private:
     template <typename T>
     static bool _IsUnique(std::shared_ptr<T> const &value) {
-        return value.unique();
```

Comment:
Your change probably doesn't introduce this as an issue but the documentation cautions against using `use_count` as a uniqueness test in multithreaded environments.

https://en.cppreference.com/w/cpp/memory/shared_ptr/use_count#Notes